pandas/tests/plotting/test_misc.py

```diff
@@ -308,3 +308,26 @@ def test_get_standard_colors_random_seed(self):
         color1 = _get_standard_colors(1, color_type='random')
         color2 = _get_standard_colors(1, color_type='random')
         assert color1 == color2
+
+    def test_get_standard_colors_no_appending(self):
+        # GH20726
+
+        # Make sure not to add more colors so that matplotlib can cycle
+        # correctly.
+        from pandas.plotting._style import _get_standard_colors
+        from matplotlib import cm
+        color_before = cm.gnuplot(range(5))
+        color_after = _get_standard_colors(1, color=color_before)
+        assert len(color_after) == len(color_before)
+
+        # Original bug report example.
+        import numpy as np
+        import pandas as pd
+
+        df = pd.DataFrame(np.abs(np.random.randn(48, 4)),
```

Comment:
Do we need the call to `np.abs` for any particular reason?

Reply:
Nope. This is just a literal copy of original SO report. Removed.